features/show_membership_application.feature

```diff
@@ -86,3 +86,14 @@ Feature: As an Admin
     Given I am logged in as "hans@random.com"
     And I am on the list applications page
     Then I should see "Du har inte behörighet att göra detta."
+
+  Scenario: Clicking the edit button on show page
+    Given I am logged in as "admin@sgf.com"
+    When I am on the application page for "Emma"
+    And I click on "Redigera ansökan"
+    Then I should be on the edit application page for "Emma"
+
+  Scenario: User does not see edit-link
+    Given I am logged in as "emma@random.com"
+    When I am on the application page for "Emma"
```

Comment:
FYI - Whether you look up and then visit the membership_application page based on the first name or email doesn't  affect the test in any way.  It only affects how you get to the right user.  (Either way -- whether you found the user by first name or email -- you then visit the membership page and then continue to actually test what you want to test.)

I think that we should clean up the tests to use just one way or the other so that we reduce the code and chance for errors.  I also think that should be part of a different PR (cleaning up/ refactoring features and steps).  

So I'll accept this and then put an entry into PT. :-)